micromasters/urls.py

```diff
@@ -52,6 +53,8 @@
         name='financial_aid_action'),
     url(r'^api/v0/financial_aid_course_price/(?P<program_id>[\d]+)/$',
         GetLearnerPriceForCourseView.as_view(), name='financial_aid_course_price'),
+    url(r'^api/v0/financial_aid_documents_sent/(?P<financial_aid_id>[\d]+)/$',
+        DocumentsSentView.as_view(), name='financial_aid_documents_sent'),
```

Comment:
Ah, I see what your concern is.  The reason this is a separate endpoint is because of permissions.  We need to have an endpoint that is restricted to staff who can see and edit financial aid applications (`FinancialAidActionView`), an endpoint for users to create an application (`FinancialAidRequestView`), an endpoint that's only for users to edit their own applications (`DocumentsSentView`), and so on.

So we can certainly change the name of `DocumentsSentView` to `FinancialAidDetailView` and make it a `RetrieveUpdateAPIView`, but to my knowledge, there isn't a need for the `.get()` view right now, since that's all handled by the dashboard API.

We could also group it with the `FinancialAidRequestView`, but the `DocumentsSentView` will require a similar, but separate, url anyway since it requires `financial_aid_id` in the URL.

We're happy to restructure it differently if you think they should be, what do you think?